files/image_config/config-setup/config-setup

```diff
@@ -403,8 +403,11 @@ boot_config()
     fi
 }
 
+# read SONiC immutable variables
+[ -f /etc/sonic/sonic-environment ] && . /etc/sonic/sonic-environment
+
```

Comment:
should we read the variables on top of this shell? At least it would help readability

Reply:
I can move to the top. The reason I put it here is that some prefers to organize the script in blocks of globals, functions, script statement. I find the above line falls int he category of statement block.